test/visibility/only/canSeeTertiaryOperator.bad

```diff
@@ -1,9 +1,9 @@
 canSeeTertiaryOperator.chpl:1: In module 'OuterModule':
 canSeeTertiaryOperator.chpl:11: error: unresolved call '+(owned Foo, owned Foo)'
-$CHPL_HOME/modules/internal/String.chpl:370: note: this candidate did not match: +(x: bufferType, y: byteIndex)
+$CHPL_HOME/modules/internal/String.chpl:391: note: this candidate did not match: +(x: bufferType, y: byteIndex)
```

Comment:
Prediff line numbers pointing into internal modules?

Reply:
This change by itself is not needed. However, this .bad has other more-than-just-line-number changes, so this one snuck in with those.